prime-router/docs/onboarding-users/senders.md

```diff
@@ -21,3 +21,44 @@ Note for content writer: If there is overlap with the previous "Pipeline Configu
 - Report endpoint - explain how to submit data (Exists in API programmer's guide)
 - History endpoint - explain how to use + example
 - Swagger API Documentation
+
+## Rhapsody Configuration
+
+Rhapsody is a health data pipeline that provides a visual interface representing various flows of health data. It uses
```

Comment:
One final thought, otherwise looks good! I wouldn't say is perfect here, since its still not secure. The only reason we used x-functions-key with rhapsody is because rhapsody didn't support oauth2.0.